src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/PreReleaseUserViewModels.cs

```diff
@@ -23,13 +23,13 @@ public class PreReleaseUserInvitePlan
         public List<string> Invitable { get; } = new List<string>();
     }
 
-    public class PreReleaseUserInviteRequest
+    public class PreReleaseUserInviteViewModel
     {
-        [Required] public List<string> Emails { get; set; } = new List<string>();
+        [EmailAddress] public List<string> Emails { get; set; } = new List<string>();
```

Comment:
Does this `[EmailAddress]` validation attribute work on a collection? Interesting!

Reply:
Missed that. I've changed it to use `MinLength` - the previous `Required` would have done nothing since it was being initialised as an empty list.